packages/app/src/runner/index.ts

```diff
@@ -323,21 +328,17 @@ async function initialize () {
   // find out if we need to continue managing viewportWidth/viewportHeight in MobX at all.
   autStore.updateDimensions(config.viewportWidth, config.viewportHeight)
 
-  // just stick config on window until we figure out how we are
-  // going to manage it
-  window.UnifiedRunner.config = config
-
   // window.UnifiedRunner exists now, since the Webpack bundle with
   // the UnifiedRunner namespace was injected by `injectBundle`.
   initializeEventManager(window.UnifiedRunner)
 
   window.UnifiedRunner.MobX.runInAction(() => {
-    const store = initializeMobxStore(window.UnifiedRunner.config.testingType)
+    const store = initializeMobxStore(window.__CYPRESS_TESTING_TYPE__)
 
     store.updateDimensions(config.viewportWidth, config.viewportHeight)
   })
 
-  window.UnifiedRunner.MobX.runInAction(() => setupRunner(config.namespace))
```

Comment:
It looks like this function wasn't using `config.namespace` for anything, this is just cleanup?

Reply:
Yep, unused.